### PR TITLE
Feat/intake

### PR DESCRIPTION
**Pull request by e-bauman**

### CODE REVIEW

File: subsystem/intake.py [hunk 1]
```diff
@@ -18,10 +18,10 @@ def __init__(self):
...
-            can_id=config.outer_intake_front_id,
-            config=OUTER_CONFIG
-        )
+        # self.outer_motor_front: SparkMax = SparkMax(
+        #     can_id=config.outer_intake_front_id,
+        #     config=OUTER_CONFIG
+        # )
 
         self.outer_motor_back: SparkMax = SparkMax(
             can_id=config.outer_intake_back_id,
```
Comment (Inspirol) [inline subsystem/intake.py:27]: Make sure to check the parameter that enables brushed motors

**Review by Inspirol — CHANGES_REQUESTED** [review]
good otherwise

**Comment by mbardoe** [comment]
@e-bauman  Here are a few notes from our code Review

Put config in the config file
Double check motor config syntax

Outer motor back and front should just be outer_motor
Delete commented code please
Line 74 creating variable is redundant, just return
What is the point of idle_speed?

**Comment by e-bauman** [comment]
idle_speed is because we are going to have the intake idly spinning when it is not actively intaking. When we don't have a note, it will idly spin in and increase in speed when we detect a note, when we do have a note, it will idly spin out so we don't accidentally pick up another note.

In the detect_note function, I am not making a new variable, I am changing the state of the subsystem variable note_in_intake.

I made the other changes from the code review

**Review by Inspirol — APPROVED** [review]
awesome work

**Review by mbardoe — APPROVED** [review]

**Pull request merged**